client/debian/apt-spacewalk/apt-spacewalk.spec

```diff
@@ -0,0 +1,150 @@
+%{!?__python2:%global __python2 /usr/bin/python2}
```

Comment:
I imagine this package needs to be SUSE-ified a bit (splitting the changelog out, adding the copyright header, etc.)...

Reply:
yes, as soon as we want to build it. For now it is ok. We just pulled in what spacewalk has.